static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -332,10 +338,27 @@ export function DisasterEventMapTile(
         Object.values(disasterEventData.provenanceInfo).forEach((provInfo) => {
           sources.add(provInfo.provenanceUrl);
         });
+        const mapPointsData = {};
```

Comment:
pending time, would be nice to split this up into smaller functions. in the meantime, add some comments?

Reply:
done, created a utils function for this